app/services/state_file/messaging_service.rb

```diff
@@ -20,10 +20,10 @@ def message_tracker
       @message_tracker ||= MessageTracker.new(data_source: data_source, message: message)
     end
 
-    def send_message
+    def send_message(use_verification=true)
```

Comment:
```suggestion
    def send_message(use_verification: true)
```
if you want to make it a named argument